docs/io.cozy.contacts.md

```diff
@@ -84,6 +84,24 @@ Used to group contacts together. A group can obviously have multiple contacts, b
 - `trashed`: {boolean} is `true` if the group is marked for removal and will be deleted soon
 - `metadata`: {object}
   - `version`: {integer} Used for migrations. Current version is **1**
+- `relationships`: {object} links between documents
+
+  - `accounts`: {object}
+    - `data`: {array} list of accounts
+      - `_id`: {string} id of the io.cozy.contacts.accounts document
+      - `_type`: {string} doctype "io.cozy.contacts.accounts"
+- `cozyMetaData`: {object}
+
+  - ...`cozyMetadata`: [Document metadata](./README.md#document-metadata)
+  - `sync`: {object}
+
+    - `ded4265b38c54b0683408c76d9ebd`: {object} id of the sourceContactsAccount
+
+      - `konnector`: {string} example : `"google"`
+      - `lastSync`: {date} (example: `"2018-10-19T10:58:37.025688+02:00"`)
+      - `contactsAccountsId`: {string} id of the io.cozy.contacts.accounts object
+      - `id`: {string} id of the remote object
```

Comment:
I would prefer `remoteId` for better understanding (not blocking)

Reply:
Yeah sorry, a bit late for that as it would also require changes to the `io.cozy.contacts` doctype and the associated connectors and apps 🤷🏼‍♂️

Reply:
Real question: Should we update contacts to have a better doctype or leave as is?

Reply:
Looping in @Crash-- and @cedricmessiant here too; I do think `remoteId` is clearer and this probably isn't too hard to change ?

Reply:
:+1: on `remoteId`. It should not be a big deal but we'll have to be careful